validator-api/src/config/mod.rs

```diff
@@ -73,6 +72,8 @@ pub struct Base {
 
     /// Address of the validator contract managing the network
     mixnet_contract_address: String,
+    // Avoid breaking derives for now
+    keypair_bs58: String,
```

Comment:
This is more of a question than a comment. Should config contain keys themselves or the paths to key files (like we're doing with mixnodes and gateways? Over there we load the .pem files based on config paths)

Reply:
I'll just leave it like this for the moment, as I don't see any reason to complicate things by reading from paths.